app/utils/importHelpers.ts

```diff
@@ -325,6 +292,48 @@ async function insertNewAccountsAndCredentials(
     }
 }
 
+/**
+ * Inserts an array of new identities and its attached accounts and credentials. The identities
+ * must be new identities that did not already exist in the database. The accounts and credentials for
+ * each identity in the array will have their identityId reference updated to point to the
+ * correct id for their identity, before they are also inserted into the database.
+ *
+ * Note that for new identities that there cannot be existing account or credentials, so we
+ * can safely insert them without checking if they already exist.
```

Comment:
This is a lie so the documentation should be updated, also because the functionality has changed from it used to do.